source/winVersion.py

```diff
@@ -218,4 +218,10 @@ def isUwpOcrAvailable():
 
 
 def isFullScreenMagnificationAvailable():
-	return getWinVer() >= WIN8
+	# Technically this is always False:

```

Comment:
This comment is not true for NVDA running on 32-bit versions of Windows, right? Has it been considered to return `False` here only if the build of Windows isn't supported and the  running version of Windows is 64-bit?

Reply:
Only if that is the cause of this issue. I would not want to include versions that have not been tested and proven to work.